src/promptflow-azure/promptflow/azure/_pf_eval_dependencies.py

```diff
@@ -0,0 +1,11 @@
+# ---------------------------------------------------------
```

Comment:
Maybe orgnize it like: so we can keep the root namepsace cleaner
_dependencies/_pf_eval.py